src/test/java/nextstep/subway/line/domain/LineRepositoryTest.java

```diff
@@ -235,10 +214,65 @@ void validateContation() {
 
 
         assertThatThrownBy(() -> {
-            line.add(section);
+            line.addSection(section);
         }).isInstanceOf(IllegalArgumentException.class);
     }
 
+    @DisplayName("[구간 삭제] 하행 종점역 삭제")
+    @Test
+    void removeSection() {
+
+        Station C역 = stationRepository.save(new Station("C역"));
+        Station A역 = stationRepository.save(new Station("A역"));
+        Line line = createLineWithUpStationAndDownStation(A역, C역);
+        Station B역 = stationRepository.save(new Station("B역"));
+        addSection(A역, line, B역);
+
+        line.removeSection(C역);
+
+        assertThat(line.getSections().getStations()).containsExactlyElementsOf(Arrays.asList(B역, A역));
+    }
+
+    @DisplayName("[구간 삭제] 역 사이 삭제")
+    @Test
+    void removeSection2() {
+
+        Station C역 = stationRepository.save(new Station("C역"));
+        Station A역 = stationRepository.save(new Station("A역"));
+        Line line = createLineWithUpStationAndDownStation(A역, C역);
+        Station B역 = stationRepository.save(new Station("B역"));
+        addSection(A역, line, B역);
+
+        line.removeSection(A역);
+
+        assertThat(line.getSections().getStations()).containsExactlyElementsOf(Arrays.asList(B역, C역));
+    }
+
+    @DisplayName("[구간 삭제] 상행 종점역 삭제")
+    @Test
+    void removeSection3() {
+        // given
+        Station C역 = stationRepository.save(new Station("C역"));
+        Station A역 = stationRepository.save(new Station("A역"));
+        Line line = createLineWithUpStationAndDownStation(A역, C역);
+        Station B역 = stationRepository.save(new Station("B역"));
+        addSection(A역, line, B역);
+        // when
+        line.removeSection(B역);
+        // then
+        assertThat(line.getSections().getStations()).containsExactlyElementsOf(Arrays.asList(A역, C역));
```

Comment:
```suggestion
        assertThat(line.getSections().getStations())
                .containsExactly(A역, C역);
```

리스트 없이 요렇게 해도 됩니다 :)

Reply:
꿀팁..!!!! 감사합니다. 변경하였습니다.